src/unix/fs.c

```diff
@@ -231,7 +231,45 @@ static ssize_t uv__fs_futime(uv_fs_t* req) {
 
 
 static ssize_t uv__fs_mkdtemp(uv_fs_t* req) {
+#if defined(__MVS__)
+  /* There is no mkdtemp. So instead use mktemp to generate a
```

Comment:
@sam-github I have modified mkdtemp as you suggested to be similar the one in src/win/fs.c.
I used /dev/urandom to fill the random buffer.